pywhat/Data/regex.json

```diff
@@ -284,7 +288,7 @@
       "Description": null,
       "Exploit": null,
       "Rarity": 1,
-      "URL": null,
+      "URL": "https://graph.facebook.com/oauth/access_token?client_id=ID_HERE&client_secret=SECRET_HERE&redirect_uri=&grant_type=client_credentials",
```

Comment:
Hey 👋 This isn't how URLs work in our program :) Per our docs:

> URL (string): If there is a way to analyse the text online, put it here. An example is Latitude & Longitude coordinates with a URL linking to Google Maps. So it'd do maps.google.com/place/. This way it takes you directly there. Not all regex have URls.

So let's say it matches on "TEST_STRING_HERE"

It'll print:

https://graph.facebook.com/oauth/access_token?client_id=ID_HERE&client_secret=SECRET_HERE&redirect_uri=&grant_type=client_credentialsTEST_STRING_HERE

Instead, you can use the `Exploit:`  key. And where it says `SECRET_HERE` replace with `AUTH_TOKEN_HERE` :)

Reply:
Yeah, got it!

Replaced now and tested with `fixtures/file`.